R/workflows.R

```diff
@@ -120,3 +120,16 @@ topological_sort <- function(graph) {
   }
   names(graph)[graph_sorted]
 }
+
+workflow_combine_status <- function(report_status) {
```

Comment:
Consider making this (and the corresponding endpoint) having a reason for the failure;i.e., which failure we have